cmake/Modules/FindLibClang.cmake

```diff
@@ -46,8 +46,8 @@ execute_process(COMMAND ${LLVM_CMD} version
     RESULT_VARIABLE llvm_config_VERSION_status
     OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-execute_process(COMMAND ${LLVM_CMD} cpp-flags
-    OUTPUT_VARIABLE llvm_config_CPPFLAGS
+execute_process(COMMAND ${LLVM_CMD} cxx-flags
+    OUTPUT_VARIABLE llvm_config_CXXFLAGS
```

Comment:
why is it necessary to rename it from cpp to cxx?
this should only have an effect on the variable name and nothing else thus no change in the behavior

Reply:
Without knowing the context I might add that CPP and CXX is usually not the same thing. Usually CPP stands for the C Pre Processor and consists of another set of flags